libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/LocaleEx.kt

```diff
@@ -13,13 +15,24 @@ import java.util.Locale
  *
  * @return unit type for specified locale
  */
-@VoiceUnit
-fun Locale.getUnitTypeForLocale(): String =
-    when (this.country) {
-        "US", // US
-        "LR", // Liberia
-        "MM" -> // Burma
-            IMPERIAL
-        else ->
-            METRIC
+object LocaleEx {
```

Comment:
We can now safely remove the `LocaleEx` from `libdirections-offboard` right?

Reply:
@Guardiola31337 Can we take care of this as a part of separate PR? I don't think we should club this change here

Reply:
yes, i'll remove it.

Reply:
> yes, i'll remove it.

It seems that @cafesilencio already removed it 🚀 😅